diff -x .DS_Store -uNr mame-0.116/src/emu/cpu/ssp1610/ssp1610.c mame/src/emu/cpu/ssp1610/ssp1610.c
--- mame-0.116/src/emu/cpu/ssp1610/ssp1610.c	2007-06-21 01:05:20.000000000 -0500
+++ mame/src/emu/cpu/ssp1610/ssp1610.c	2007-06-21 02:11:29.000000000 -0500
@@ -136,7 +136,7 @@
 static int ssp1610_execute(int cycles)
 {
 	ssp1610_ICount = cycles;
-
+#if 0
 	do
 	{
 		UINT16 op;
@@ -663,7 +663,7 @@
 		ssp1610_ICount--;
 
 	} while( ssp1610_ICount > 0 );
-
+#endif
 	return cycles - ssp1610_ICount;
 }
 
